.github/workflows/docker.yml

```diff
@@ -37,6 +40,12 @@ jobs:
           registry: ghcr.io
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
+      -
+        name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
```

Comment:
Are these actions required for us to publish the image into DockerHub?

Reply:
Yes, all the docs I found on linking to the Dockerhub account lead to this.

@vraravam if you have access to this repo secrets maybe you can please message me on discord so I can send you the Dockerhub username and an access token.

Reply:
i thought you had access as a Maintainer - let me check your role @SpecialAro